pom.xml

```diff
@@ -94,7 +86,6 @@
                     <loggers>
                         <hudson.plugins.audit_trail>FINE</hudson.plugins.audit_trail>
                     </loggers>
-                    <minimumJavaVersion>8</minimumJavaVersion>
```

Comment:
Obsoleted by a previous parent POM upgrade when Java 11 or later began to be a requirement.